components/brave_rewards/resources/ui/internals/reducers/rewards_internals_reducer.ts

```diff
@@ -28,6 +28,7 @@ const rewardsInternalsReducer: Reducer<RewardsInternals.State | undefined> = (st
       state.isRewardsEnabled = action.payload.enabled
       break
     case types.ON_GET_REWARDS_INTERNALS_INFO:
+      console.log(action.payload)
```

Comment:
This is intentional or leftover debug logging?

Reply:
left over, removed